third-party/easyloggingpp/src/easylogging++.cc

```diff
@@ -2398,11 +2427,30 @@ void AsyncDispatchWorker::handle(AsyncLogItem* logItem) {
 #  endif  // defined(ELPP_SYSLOG)
 }
 
+// This method is used in order to transfer all the logs:
+// from the "write queue" - queue in which all the logs are added by the other threads
+// to the "read queue" - queue from which the logs are read and dispatched by the async logger's thread
+// This double buffer mechanism avoids from the other threads the need to wait writing their logs 
```

Comment:
// This double buffer mechanism minimizes the inter-thread locking time, improving log's bandwidth and preventing costly stalls due to log flushes to HD.

Reply:
done